lib/emulationHandler.js

```diff
@@ -9,12 +9,12 @@ const setViewport = async(options) => {
         throw new Error('No height and width provided');
     options.mobile = options.mobile || false;
     options.deviceScaleFactor = options.deviceScaleFactor || 1;
+    options.screenOrientation = options.screenOrientation || options.isLandscape ? { angle: 90, type: 'landscapePrimary' } : { angle: 0, type: 'portraitPrimary' };
```

Comment:
will be great to update documentation with options available

Reply:
@NivedhaSenthil Its already linked 

https://github.com/getgauge/taiko/blob/master/lib/taiko.js#L217

Reply:
👍  I was referring to `emulateDevice` docs as the title suggests and just realised it does not take any options